src/com/obelikova/console/calculator/Operations.java

```diff
@@ -0,0 +1,34 @@
+package com.obelikova.console.calculator;
+
+public class Operations {
+  private double operand1;
+  private double operand2;
+  private double result;
+  private double operation;
+
+  public Operations() {}
+
+  public double add(double operand1, double operand2) {
```

Comment:
All the methods might be changed to be static ones: there is no need to instanciate the class to work with it's API